### ISSUE TITLE
JSON Downloads wont work

### ISSUE THREAD

**Issue by burner03** [bigbass1997/NotEnoughProduction#29]
Hey Bigbass, me and my friends just came back to GTNH and remembered how useful this app was to our process. 

Is the web server down for the json? Could it be uploaded to somewhere like mega or on this repo?

**Comment by bigbass1997** [comment]
Yeah sorry about that. If you want the old releases, I'll figure out another place to keep what I have for you. Instead I recommend using RecEx to get the json exports yourself from the game. https://github.com/bigbass1997/RecEx/releases/tag/v0.0.3

**Comment by burner03** [comment]
So I got the RecEx export (looks good) but unfortunately the app looks for the md5 hash of the files on the server. I guess I can compile the app and remove that check though. Thanks for getting back to me & the work done here, it's great.

**Comment by bigbass1997** [comment]
Hmm thought it would continue if it didn't find it there. But honestly I don't remember for sure. I'm not working on this anymore, moved onto other projects unrelated to MC.

Shouldn't be too hard to disable the hash checking like you said though. Let me know if you have any problems compiling.
